x/post/types/events.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	linotypes "github.com/lino-network/lino/types"
+)
+
+// RewardEventV1 - used only for importing.
```

Comment:
is it possible to export without Original and Friction fields so v1 can be removed?

Reply:
The problem is the type of `	Evaluate   linotypes.Coin       `json:"evaluate"`  changed, and in v1 we don't have minidollar.